pkg/util/cmd_utils.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2017 Google, Inc. All rights reserved.
```

Comment:
A slightly better name/location for this might be "cmd/util/output/output.go" or something, so it's imported as "output".